extensions/radial_profile_spotfinder_threshold_ext.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import annotations
+
+from scitbx.array_family import flex
+
+import dials.extensions
+from dials.algorithms.image.filter import convolve
+
+
+class RadialProfileSpotFinderThresholdExt:
+    """
+    Extension to calculate a radial profile threshold. This method calculates
+    background value and sigma in 2θ shells, then sets a threshold at a level
+    n_sigma above the radial background. As such, it is important to have the
+    beam centre correct and to mask out any significant shadows. The method may
+    be particularly useful for electron diffraction images, where there can be
+    considerable inelastic scatter around low resolution spots. In addition, the
+    algorithm is relatively insensitive to noise properties of the detector.
+    This helps for the case of integrating detectors with poorly known gain
+    and response statistics.
+
+    A similar algorithm is available in other programs. The description of
+    'peakfinder 8' in https://doi.org/10.1107/S1600576714007626 was helpful
+    in the development of this method.
+    """
+
+    name = "radial_profile"
+
+    @staticmethod
+    def phil():
+        from libtbx.phil import parse
```

Comment:
Is there a specific reason for the inline import here?

Reply:
Copied from `DispersionSpotFinderThresholdExt`. I'll float them both

Reply:
There _might_ be a valid reason for doing this for extensions. @ndevenish might anyway look at running an auto-float script across `dials/`. I'll leave it as-is here then.

Reply:
Yes, I think the floating scripts were never run on DIALS. It probably should at some point.